### PR TITLE
Added 'enable_security: true | false'

### PR DESCRIPTION
**Pull request by arghav**
The idea is to allow the use of the @anywhere templating helper without configuring security.

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -19,6 +19,7 @@ class Configuration{
 
         $rootNode
             ->scalarNode('file')->isRequired()->cannotBeEmpty()->end()
+            ->scalarNode('enable_security')->defaultValue(false)->end()
```
Comment (ornicar) [inline DependencyInjection/Configuration.php]: I think this should be

```
->booleanNode('enable_security')->defaultFalse()->end()
```

File: DependencyInjection/FOSTwitterExtension.php [hunk 2]
```diff
@@ -21,6 +20,10 @@ class FOSTwitterExtension extends Extension
         $configuration = new Configuration();
         $config = $processor->process($configuration->getConfigTree(), $configs);
 
+        if($config['enable_security']){
```
Comment (ornicar) [inline DependencyInjection/FOSTwitterExtension.php]: According to symfony CS, this line is missing two spaces:

```
if ($config['enable_security']) {
```

File: DependencyInjection/Configuration.php (hunk 1 above)
Comment (lsmith77) [inline DependencyInjection/Configuration.php]: yeah both work .. but ornicar's version is cleaner.

Comment (arghav) [inline DependencyInjection/Configuration.php]: yeah right, fixed in the new commit

File: DependencyInjection/FOSTwitterExtension.php (hunk 2 above)
Comment (arghav) [inline DependencyInjection/FOSTwitterExtension.php]: fixed in the new commit

**Comment by lsmith77** [comment]
discussed this on IRC .. the better solution would be to fix the DI to not complain in case an abstract definition is missing a parent until the abstract definition is inherited. just someone needs to write that fix now ...

**Comment by arghav** [comment]
Closing this PR as the fix has been added to the core. https://github.com/symfony/symfony/pull/210

**Pull request closed without merging**